inference-engine/src/inference_engine/generic_ie.cpp

```diff
@@ -133,8 +133,7 @@ void ngraph::op::GenericIE::validate_and_infer_types() {
         }
 
         // WA: shape infer has to know number of outputs
-        if ((type == "ExperimentalDetectronROIFeatureExtractor" || type == "ExperimentalDetectronDetectionOutput")
```

Comment:
Update copyrights year in all files you modify.

Reply:
Fixed.